ingest/expression_files/mtx.py

```diff
@@ -75,6 +82,26 @@ def check_valid(
             raise ValueError("; ".join(error_messages))
         return True
 
+    @staticmethod
+    def is_sorted(file_path: str):
+        """Checks if a file is sorted by gene index"""
+        file_size = os.path.getsize(file_path)
+        if file_size == 0:
+            raise ValueError(f"{file_path} is empty: " + str(file_size))
+        start_idx: int = MTXIngestor.get_line_no(file_path)
```

Comment:
this needs a high level comment about why we need to use tail rather than just reading in the file directly by sort

Reply:
464ec68